style.css

```diff
@@ -221,6 +221,18 @@ i.hamburger {
   color: #172b4d;
 }
 
+footer span {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #4252;
```

Comment:
Kindly change this color according to Figma design to make the email in your footer visible 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187037826-6bf2adf2-2e5d-4e90-9d28-001d7275e993.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187037833-4f0613bc-d120-417d-bd12-71c128edd03f.png)

Reply:
done.